Automattic/jetpack
# VideoPress: Set the filename for VideoPress attachments

Fixes # 989-gh-Automattic/greenhouse

The filename has been missing from the VideoPress video attachments in Media Library, and an empty div has been displayed below those videos.

File size is also missing, however it will be addressed separately because the solution is a little more involved.

**Before**
<img width="488" alt="Screen Shot 2021-10-28 at 3 07 29 PM" src="https://user-images.githubusercontent.com/4081020/139319876-2d8e3040-ec3f-4002-ac8c-d45cdd706869.png">
<img width="229" alt="Screen Shot 2021-10-28 at 3 09 22 PM" src="https://user-images.githubusercontent.com/4081020/139320181-67f224eb-eefd-4958-9b9c-d39785e25173.png">


**After**
<img width="485" alt="Screen Shot 2021-10-28 at 3 02 56 PM" src="https://user-images.githubusercontent.com/4081020/139319606-5c921cbd-5fce-41ba-bcf2-b3b10f2e2788.png">
<img width="221" alt="Screen Shot 2021-10-28 at 3 03 05 PM" src="https://user-images.githubusercontent.com/4081020/139319612-90773d77-43a2-4abb-949c-17b02d5cebd9.png">

#### Changes proposed in this Pull Request:

* Adds a filter for `get_attached_file` so the filename can be displayed in the Media Library

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


Nope

#### Testing instructions:






* Apply the PR
* Go to a site with VideoPress enabled and videos already uploaded, the file name should appear in the media library, and in the media item view when a video is opened
* The `File URL` field in the Media Item view should still include the `videos.files.wordpress.com/<guid>/<filename>` video url
